angular/src/app/app.component.ts

```diff
@@ -1,57 +1,33 @@
-import { Component } from '@angular/core';
-
-// Calcite Components
-import '@esri/calcite-components/dist/components/calcite-icon';
-import '@esri/calcite-components/dist/components/calcite-combobox';
-import '@esri/calcite-components/dist/components/calcite-combobox-item';
-
+import { Component } from "@angular/core";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
 })
 export class AppComponent {
-  title: string = "Angular";
+  title = "calcite-components-angular-example";
+  sliderValue = 50;
 
-  fields = [
-    {
-      "displayText": "Natural Resources",
-      "value": "natural-resources"
-    },
-    {
-      "displayText": "Agriculture",
-      "value": "agriculture"
-    },
-    {
-      "displayText": "Forestry",
-      "value": "forestry"
-    },
-    {
-      "displayText": "Mining",
-      "value": "mining"
-    },
-    {
-      "displayText": "Business",
-      "value": "business"
-    },
-    {
-      "displayText": "Education",
-      "value": "education"
-    },
-    {
-      "displayText": "Utilities",
-      "value": "utilities"
-    },
-    {
-      "displayText": "Transportation",
-      "value": "transportation"
-    }
-  ];
+  public isLoading: boolean = true;
+
+  ngOnInit() {
```

Comment:
The components do cleanup internally in [`disconnectedCallback`](https://stenciljs.com/docs/component-lifecycle#disconnectedcallback), but we don't expose any cleanup methods to users. Is that what you meant?

Reply:
It might answer my question, mainly I'm asking about the potential for memory leaks. I guess if the Calcite components do their own internal cleanup that should remove all Calcite component references when the "parent" Angular component shuts down. 

We seen these usage patterns with the JS SDK where users open and close Angular components many times during their daily workflow. Behind the scenes, this creating/destroying of Angular components can cause memory leaks which result in app crashes.

As an example, here's what we do with MapView:

```js

  ngOnDestroy(): void {
    if (this.view) {
      // destroy the map view
      this.view.destroy();
    }
  }
```

Reply:
Gotcha, yeah we do all the stuff like disconnecting MuationObservers internally. But please log an issue if you do see or hear something about memory leaks!